tx/textual/internal/testdata/coin.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
> Stacked on top of https://github.com/cosmos/cosmos-sdk/pull/12729

I propose to not stack my `Coin` commits in this PR. So only 5bd2cefd2fb6e261b353d36d21ddc1108b193bda is needed for this PR.

This way we can probably merge this PR before the `Coin` one (which still needs some work).

Reply:
Right on, lemme rebase..